src/Routes/RoutesCommon.tsx

```diff
@@ -180,27 +180,27 @@ export const routes: NestedRoute[] = [
         main: TransactionHistory,
         category: RouteCategory.Wallet,
       },
+      ...assetIdPaths.map(assetIdPath => ({
+        path: `/asset${assetIdPath}`,
+        label: 'Overview',
+        icon: <AssetsIcon />,
+        main: Asset,
+        hide: true,
+      })),
     ],
   },
   {
-    path: '/assets',
-    label: 'navBar.assets',
+    path: '/markets',
+    label: 'navBar.markets',
     main: Assets,
     icon: <AssetsIcon />,
     category: RouteCategory.Explore,
     routes: assetIdPaths.map(assetIdPath => ({
-      path: assetIdPath,
+      path: `/asset${assetIdPath}`,
```

Comment:
@reallybeard that seems off, an asset page isn't a market page. What's the rationale behind this?

Reply:
Its a routing thing to keep the menu highlighted and follow you down the path. They are the same component, but you get there a different way. 

This will be useful for the mobile app, because you don't usually jump around between top level routes in a mobile app, its a structure page -> sub page.

So if you are looking at the home screen, and tap into an asset. You are now one level deep on the home page.

And if you tap on an asset from my wallet you are now one level deep on wallet for example.

Home > Bitcoin
Wallet > Bitcoin

So if the user navigates to these routes, they will remember the context of how they got there.